src/common/transformations/src/transformations/op_conversions/convert_interpolate1_to_interpolate4.cpp

```diff
@@ -44,20 +44,7 @@ ngraph::pass::ConvertInterpolate1ToInterpolate4::ConvertInterpolate1ToInterpolat
         if (attrsV0.mode == "nearest") {
             attrsV4.mode = ngraph::opset4::Interpolate::InterpolateMode::NEAREST;
         } else if (attrsV0.mode == "linear") {
-            // If we write only
-            //    attrsV4.mode = ngraph::op::v4::Interpolate::InterpolateMode::linear;
-            // instead of a conditional statements below when attrsV0.mode == "linear",
-            // then we have a performance drop, because CPU have no optimized
-            // version of the 'linear' mode.
-            // TODO: delete this conditional statement, when CPU will have
-            // optimized version of the 'linear' mode.
-            if (input_shape_rank < 5) {
-                attrsV4.mode = ngraph::op::v4::Interpolate::InterpolateMode::LINEAR_ONNX;
-            } else if (input_shape_rank == 5) {
-                attrsV4.mode = ngraph::op::v4::Interpolate::InterpolateMode::LINEAR;
-            } else {
-                return false;
-            }
+            attrsV4.mode = ngraph::op::v4::Interpolate::InterpolateMode::LINEAR;
```

Comment:
We have to make sure that newly optimized version works at least not slower than LINEAR_ONNX.
@chenhu-wang May I ask you to help here with proper benchmarking?

Reply:
@dmitry-gorokhov  talked online meeting, will do this.